roles/nginx/defaults/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+nginx_version: '1.18*'
```

Comment:
In the `docs/vars.md` you wrote that the default version is 1.19. Shouldn't it be too?